MOONCLOCK/apps.py

```diff
@@ -144,10 +147,10 @@ def update(self, first, remaining_duration):
 
         self.display_group.clear()
         self.display_group.render_string(
-            '{0}{1}{2}'.format(
+            '{0}{1}{2} '.format(
```

Comment:
remove concatenation

Reply:
If the wide character is the last character in the string, you don't need to add a space.

Reply:
Currently, this is needed otherwise the logo is cut off for some reason.  I'll open an issue for this bug.